ee/ucp/admin/configure/create-audit-logs.md

```diff
@@ -4,64 +4,115 @@ description: Learn how to create audit logs of all activity in UCP
-- **Security Analysis and Auditing** - Security is one of the primary uses for audit logs. A full record of all user interactions with the container infrastructure gives your security team full visibility to questionable or attempted unauthorized accesses.
-- **Chargeback** - You can use audit logs and information about the resources to generate chargeback information.
-- **Alerting** - If there is a watch on an event stream or a notification created by the event, alerting features can be built on top of event tools that generate alerts for ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
+- **Historical Troubleshooting** - Audit logs are helpful in determining a 
+sequence of past events that explain why an issue occured.
+
+- **Security Analysis and Auditing** - Security is one of the primary uses for 
+audit logs. A full record of all user interactions with the container 
+infrastructure gives your security team full visibility to questionable or 
+attempted unauthorized accesses.
+
+- **Chargeback** - You can use audit logs and information about the resources to 
+generate chargeback information.
+
+- **Alerting** - If there is a watch on an event stream or a notification 
+created by the event, alerting features can be built on top of event tools that 
+generate alerts for ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
+
+## Enablig UCP Audit Logging
+
+UCP audit logging can be enabled via the UCP web user interface or via the
+UCP configuration file.
+
+### Enabling UCP Audit Logging via UI
+
+1) Log in to the **UCP** Web User Interface
+
+2) Navigate to **Admin Settings**
 
-## Procedure
+3) Select **Audit Logs**
 
-1. Download the UCP Client bundle [Download client bundle from the command line](https://success.docker.com/article/download-client-bundle-from-the-cli).
+4) In the **Configure Audit Log Level** section, select the relevant logging
+level. 
 
-2.  Retrieve JSON for current audit log configuration.
+![Enabling Audit Logging in UCP](../../images/auditlogging.png){: .with-border}
 
-    ```
-    export DOCKER_CERT_PATH=~/ucp-bundle-dir/
-    curl --cert ${DOCKER_CERT_PATH}/cert.pem --key ${DOCKER_CERT_PATH}/key.pem --cacert ${DOCKER_CERT_PATH}/ca.pem -k -X GET https://ucp-domain/api/ucp/config/logging > auditlog.json
-    ```
+5) Click **Save**
 
-3. Open auditlog.json to modify the 'auditlevel' field to `metadata` or `request`.
+### Enabling UCP Audit Logging via Config File
 
-    ```
-    {
-    	"logLevel": "INFO",
-    	"auditLevel": "metadata",
-    	"supportDumpIncludeAuditLogs": false
-    }
-    ```
+Enabling UCP audit logging via the UCP Configuration file can be done before 
+or after a UCP installation. Following the UCP Configuration file documentation 
+[here](../ucp-configuration-file/).
 
-4. Send the JSON request for the auditlog config with the same API path but with the `PUT` method.
+The section of the UCP configuration file that controls UCP auditing logging is:
 
-    ```
```

Comment:
We should still leave this in actually as an alternative to doing this over the CLI